.github/workflows/release.yml

```
@@ -0,0 +1,63 @@
+name: Release
+
+on:
+  push:
+    branches: main
+  workflow_dispatch:
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    env:
+      SHA: ${{ github.sha }}
+      GH_TOKEN: ${{ inputs.github-token }}
```

Unless I'm missing something, I think you meant

```suggestion
      GH_TOKEN: ${{ github.token }}
```